lms/static/js/views/fields.js

```diff
@@ -18,9 +20,17 @@
 
         var messageRevertDelay = 6000;
         var FieldViews = {};
+        var statusButtonHtml = function(label, classNames) {
+            return HtmlUtils.joinHtml(
+                HtmlUtils.HTML('<i class="fa ' + classNames + '" aria-hidden="true"></i>'),
```

Comment:
The linter may look for this.  It's fine either way, but ultimately it may require a linter disable pragma.